src/main/java/mate.academy/service/impl/ProductServiceImpl.java

```diff
@@ -2,13 +2,17 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 import mate.academy.service.FileReaderService;
 import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
+@Component
 public class ProductServiceImpl implements ProductService {
+    @Component
     private ProductParser productParser;
+    @Component
```

Comment:
```suggestion
    @Inject
```
Inject is used above the field you want to be initialized

Reply:
done